app/src/main/java/com/calculator/CalcFragment.kt

```diff
@@ -0,0 +1,117 @@
+package com.calculator
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import com.calculator.CalcLogic.parse
+import com.calculator.databinding.FragmentCalcBinding
+import com.example.homework.Calculator
+
+
+class CalcFragment : Fragment(R.layout.fragment_calc) {
```

Comment:
если используется viewbinding, то ты вью создаешь в onCreateView, можно использовать обычный конструктор 
```suggestion
class CalcFragment : Fragment() {
```